packages/open-next/src/adapters/routing/matcher.ts

```diff
@@ -180,6 +180,7 @@ export function handleRedirects(
       type: event.type,
       statusCode: 308,
       headers: {
+        ...event.headers,
```

Comment:
Unless i'm missing something, we don't want to pass `event.headers` here.
`handleRedirects` are for `next.config.js` redirects and `event.headers` in this case are request headers, not response headers.